config/rabbitmq.yml

```diff
@@ -15,14 +16,11 @@ test:
   exchange: published_documents_test
 
 production:
+  <<: *defaults
```

Comment:
This object will have both as `host` key (set to `localhost`) and a `hosts` key set to the RabbitMQ hosts - will that cause any problems?

Reply:
Good question. I was hoping at the time that it would just use `hosts` if specified. I've checked though, and it at least checks if both have been given, and warns against this, so I've moved it out of the defaults now.